tools/src/bin/build-blame.rs

```diff
@@ -278,15 +277,15 @@ fn blame_for_path(
 ) -> Result<String, git2::Error> {
     let linecount = count_lines(&blob);
     // TODO: drop this author field entirely, I don't think it gets consumed by anything
```

Comment:
This TODO line can be removed, can't it?

Reply:
Oh indeed, yes it can :)

Reply:
https://github.com/mozsearch/mozsearch/pull/431

Reply:
thanks!